client/blocks/jetpack-benefits/index.tsx

```diff
@@ -0,0 +1,184 @@
+		isJetpackScanSlug( productSlug ) ||
+		// check plans for Jetpack scan features
+		( isJetpackPlanSlug( productSlug ) &&
+			( planHasFeature( productSlug, FEATURE_JETPACK_SCAN_DAILY ) ||
+				planHasFeature( productSlug, FEATURE_JETPACK_SCAN_DAILY_MONTHLY ) ) )
+	);
+};
+
+export const productHasSearch = ( productSlug: string ) => {
+	return (
+		// standalone search product
+		// there is not currently a isJetpackSearchSlug
+		JETPACK_SEARCH_PRODUCTS.includes( productSlug ) ||
+		// check plans for Jetpack search features
+		( isJetpackPlanSlug( productSlug ) &&
+			( planHasFeature( productSlug, FEATURE_JETPACK_SEARCH ) ||
+				planHasFeature( productSlug, FEATURE_JETPACK_SEARCH_MONTHLY ) ||
+				// This is a bit obscure - checks specifically for Jetpack Business (Professional)
+				// Is it an error that the plan spec in plans-list.js does not contain search features?
+				planHasFeature( productSlug, FEATURE_ALL_PREMIUM_FEATURES_JETPACK ) ) )
+	);
+};
+
+export const productHasAntiSpam = ( productSlug: string ) => {
+	// check that this product is standalone anti-spam or one of the plans that contains it
+	return (
+		// standalone anti-spam product
+		isJetpackAntiSpamSlug( productSlug ) ||
+		// check plans for anti-spam features
+		( isJetpackPlanSlug( productSlug ) &&
+			( planHasFeature( productSlug, FEATURE_JETPACK_ANTI_SPAM ) ||
+				planHasFeature( productSlug, FEATURE_JETPACK_ANTI_SPAM_MONTHLY ) ) )
+	);
+};
+
+export const productHasActivityLog = ( productSlug: string ) => {
+	return isJetpackPlanSlug( productSlug ) || isJetpackBackupSlug( productSlug );
+};
+
+interface Props {
+	siteId: number;
+	productSlug: string;
+}
+
+// named export for cleaner testing of class methods
+const JetpackBenefits: React.FC< Props > = ( { siteId, productSlug } ) => {
+	const rewindState = useSelector( ( state ) => {
+		return getRewindState( state, siteId );
+	} );
+
+	const scanState = useSelector( ( state ) => {
+		return getSiteScanState( state, siteId );
+	} );
+	const translate = useTranslate();
+
+	const siteHasBackups = () => {
+		return 'unavailable' !== rewindState?.state;
+	};
+
+	const siteHasScan = () => {
+		return 'unavailable' !== scanState?.state;
+	};
+
+	const renderSiteSearch = () => {
+		return (
+			<JetpackBenefitsCard
+				headline={ translate( 'Search' ) }
+				description={ translate(
+					'Jetpack Search helps your visitors instantly find the right content – right when they need it.'
+				) }
+			/>
+		);
+	};
+
+	const renderSiteAntiSpam = () => {
+		return (
+			<JetpackBenefitsCard
+				headline={ translate( 'Anti-spam' ) }
+				description={ translate(
+					'Jetpack anti-spam automatically clears spam from comments and forms.'
+				) }
+			/>
+		);
+	};
+
+	const renderSiteActivity = () => {
+		return (
+			<JetpackBenefitsCard
+				headline={ translate( 'Activity Log' ) }
+				description={ translate(
+					'The Activity Log shows a list of management events that have occurred on your site.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'The activity log shows a full list of management events that have occurred on your site.' )` *ES Score: 8*